cli/dcoscli/package/main.py

```diff
@@ -421,7 +423,7 @@ def _list(json_, app_id, package_name):
     if app_id is not None:
         app_id = util.normalize_app_id(app_id)
     results = package.installed_packages(
-        package_manager, app_id, package_name)
+        package_manager, app_id, package_name, not cli_only)
 
     # only emit those packages that match the provided package_name and app_id
     if results or json_:
```

Comment:
Question about the error message printed when there are no results (expand the code below): the message is a bit inaccurate, because if flags like `--cli` or `--app-id` are provided, there could be installed packages that aren't shown. Would it be worthwhile to have a different message when results are filtered out due to flags?